jsk_perception/node_scripts/tile_image.py

```diff
@@ -70,15 +80,19 @@ def callback(msg):
     def _append_images(self, imgs):
         if not imgs:
             return
+        # convert tile shape: (Y, X) -> (X, Y)
+        shape_xy = self._shape[::-1] if self._shape else self._shape
```

Comment:
@wkentaro What do you mean about `else self._shape`? If you mean `None` here, I think using `None` is easier to understand.